From e9ac5233f92441eb854688500c471911dfdc2ff1 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Sat, 14 Mar 2015 00:01:49 +0000
Subject: Don't build the compilation date and time into the binary

This made the build unreproducible.

Last-Update: 2015-09-26

Patch-Name: reproducible.patch
---
--- a/main/options.c
+++ b/main/options.c
@@ -1218,7 +1218,6 @@
 	printf ("Universal Ctags is derived from Exuberant Ctags.\n");
 	printf ("Exuberant Ctags 5.8, Copyright (C) 1996-2009 Darren Hiebert\n");
 
-	printf ("  Compiled: %s, %s\n", __DATE__, __TIME__);
 	printf ("  URL: %s\n", PROGRAM_URL);
 
 	printFeatureList ();
